packages/core/admin/admin/src/content-manager/components/BlocksEditor/Toolbar/index.js

```diff
@@ -104,21 +104,6 @@ ModifierButton.propTypes = {
   }).isRequired,
 };
 
-const isBlockActive = (editor, matchNode) => {
-  const { selection } = editor;
-
-  if (!selection) return false;
-
-  const match = Array.from(
-    Editor.nodes(editor, {
-      at: Editor.unhangRange(editor, selection),
```

Comment:
@remidej has another solution to get anchor node using this method. I think it will be worth checking instead of removing this method completely as its used in other places as well, for eg isActiveLink

Reply:
thanks a lot for the comment @madhurisandbhor I commented the PR https://github.com/strapi/strapi/pull/18195/files#r1342465907 and suggested to find another place for the function isBlockActive and also another name because in my PR is unused